src/containers/gui.jsx

```diff
@@ -55,16 +56,16 @@ class GUI extends React.Component {
         }
     }
     render () {
-        if (this.props.loadingError) {
+        if (this.props.isError) {
             throw new Error(
-                `Failed to load project from server [id=${window.location.hash}]: ${this.props.errorMessage}`);
+                `Failed to load project from server [id=${window.location.hash}]: ${this.props.errStr}`);
```

Comment:
I see where this change came from but it makes me sad because `errorMessage` is clearer. `errStr` looks like a temporary variable name not a permanent part of the state.  Would it be too much to ask to update `errStr` to be called `errorMessage` everywhere it's defined/used?

Reply:
Not too much to ask! I agree. (I only used a different name when creating errStr because I wasn't yet replacing it everywhere, and they had to coexist!)